modules/olm-installing-operators-from-operatorhub-configure.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * operators/user/olm-installing-operators-in-namespace.adoc
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+//
+// Module watched for changes by Ecosystem Catalog team:
+// https://projects.engineering.redhat.com/projects/RHEC/summary
+
+
+[id="olm-installing-operators-from-operatorhub-configure_{context}"]
+= Configuring {product-title} to use Red Hat Operators
+
+In {product-title}, Red Hat Operators are not available by default. You can access and install these Operators if you have a pull secret from the Red Hat OpenShift Cluster Manager site by editing the Operator Hub custom resource (CR). 
+
+.Prerequisites
+
+* Ensure that you have downloaded the link:https://cloud.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site] as shown in the installation procedure.
+
+.Procedure
+
+To access the Red Hat Operators in a {product-title} cluster: 
+
+. Edit the Operator Hub CR using the web console or CLI:
+
+.. Using the CLI:
+
+... Edit the `OperatorHub` CR object:
```

Comment:
Yes to "`OperatorHub` CR" here, but no to "object" because I think it is somewhat redundant. My understanding is that this would only be either a CR or an object but not both. Also seems that "object" can be interchangeable with the generic term "resource". Not confusing at all. 😛  

From docs_guidelines.adoc:
> Use "object", "resource", "custom resource", "spec", etc. as appropriate after the object reference. This helps with clarity and readability.